client/components/data/current-theme/index.js

```diff
@@ -65,5 +64,5 @@ export default connect(
 			currentTheme: getCurrentTheme( state, props.site.ID )
 		}
 	),
-	bindActionCreators.bind( null, { fetchCurrentTheme } )
+	{ fetchCurrentTheme }
```

Comment:
https://github.com/rackt/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options:

> * [`mapDispatchToProps(dispatch, [ownProps]): dispatchProps`] \(*Object* or *Function*): If an object is passed, each function inside it will be assumed to be a Redux action creator. An object with the same function names, but bound to a Redux store, will be merged into the component’s props.

So `bindActionCreators` is apparently redundant here and can be safely dropped, since `connect()` does the binding for us.